bin/postinstall

```diff
@@ -40,6 +40,18 @@ ${CLI} config:set SECRET_TOKEN="$secret_token"
 # migrate
 ${CLI} run rake ${rake_commands} || true
 
+# Avoid errors on versions < 5.0
+if [ -e "${APP_HOME}/lib/tasks/scm.rb" ]; then
+	# migrate previous repositories with reposman to managed
+	SVN_REPOSITORIES = $(${CLI} config:get SVN_REPOSITORIES || echo "")
```

Comment:
@oliverguenther this line does not look like valid bash. Are you sure this works?